resources/enclosure.rb

```diff
@@ -32,3 +34,20 @@
 action :remove do
   remove
 end
+
+action :reconfigure do
+  item = load_resource
+  item.retrieve!
+  item.configuration
+end
+
+action :refresh do
```

Comment:
For `:reconfigure` action it is possible to check the `enclosure['state']` before and do it only if it is `inconsistent`.

But for the `:refresh` action there is no flag indicating it needs to be refreshed. We cannot check if there are changes to do the refresh because the refresh is the action that does it itself.